src/utils/filterFilms.ts

```diff
@@ -0,0 +1,46 @@
+import { FILTERS } from "../types/filterEnum";
+import { Language } from "../types/langType";
+import { IMovie } from "../types/movie";
+import { SORT } from "../types/sortEnum";
+
+interface IParams {
+  movies: IMovie[];
+  filter: FILTERS;
+  sort: SORT;
+  lang: Language;
+}
+
+export const filterMovies = ({
+  movies,
+  filter,
+  sort,
+  lang,
+}: IParams): IMovie[] => {
+  let movieCopy = [...movies];
+
+  if (filter !== FILTERS.ALL) {
+    movieCopy = movieCopy.filter((movie) => movie.format === filter);
+  }
+
+  if (sort) {
+    movieCopy.sort((a, b) => {
```

Comment:
you can just add the next rows
const title = lang === 'en' ? a.title_en : a.title_ua 
You will remove this swithc

Reply:
done